web/Dockerfile.dev

```diff
@@ -0,0 +1,13 @@
+FROM node:18-alpine
+
+WORKDIR /app
+
+COPY package*.json .
+
+RUN npm ci
```

Comment:
You can add --audit false and --fund false for extra performance and readability